js/templates/search/search.html

```diff
@@ -46,30 +46,31 @@ <h3 class="txUnl rowSm"><%= ob.name %></h3>
         <div class="flexColWide gutterV">
           <% _.each(ob.options, function(val, key) { %>
             <div class="contentBox pad clrP clrBr clrSh2">
-              <h4 class="rowSm"><%= val.label %></h4>
+              <div class="rowSm txB tx5b"><%= val.label %></div>
               <% if (val.type ==='dropdown') { %>
                 <select name="<%= key %>" class="select2Small">
-                <% _.each(val.options, function(option) {
-                  let selected = '';
-                  if (ob.filterVals.hasOwnProperty(key)) {
-                    selected = option.value === ob.filterVals[key] ? 'selected' : '';
-                  } else {
-                    selected = option.default ? 'selected' : '';
-                  }
-                  print(`<option ${selected} value="${option.value}">${option.label}</option>`);
-                }); %>
+                <%
+                  // if any option has a selected value use the first one. Otherwise use the first default.
+                  let selectedIndex = val.options.findIndex(opt => opt.checked);
+                  selectedIndex = selectedIndex === -1 ? val.options.findIndex(opt => opt.default) : selectedIndex;
+                  _.each(val.options, function(option, ind) {
+                    const selected = selectedIndex === ind ? 'selected' : '';
+                    // parsing the label for emojis isn't needed here because the select list is replaced by selectize.js
```

Comment:
Actually, selectize is used for tagging fields, but we're still using select2 for regular dropdowns and that's what this view uses for this dropdown.

Also, in the Search.js, I noticed that select2 is not declared as a dependency, but should be. The reason it's not bombing is because some other module imported it, which added it to the jQuery object, but we shouldn't depend on that.